app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,37 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            openLink(new URI("https://github.com/CIRDLES/Topsoil/wiki"));
+        } catch (URISyntaxException e) {
+            LOGGER.error(null, e);
+        }
+    }
+
+    @FXML
+    void openHelpLink() {
+        try {
+            openLink(new URI("https://github.com/CIRDLES/Topsoil/issues"));
+        } catch (URISyntaxException e) {
+            LOGGER.error(null, e);
+        }
+
+    }
+
+    void openLink(URI uri) {
+
+        Desktop desktop = Desktop.isDesktopSupported() ? Desktop.getDesktop() : null;
+        if (desktop != null && desktop.isSupported(Desktop.Action.BROWSE)) {
```

Comment:
These lines will be hard to read when you or I look back on this code in a month.